test/ci/backend/lava.json

```diff
@@ -0,0 +1 @@
 2, "testdata": 3, "yaml_line": null, "description_line": null, "log_section": null, "duration": null, "timeout": 30, "testcase": null, "count": null, "max_retries": 1}}, {"model": "lava_results_app.actiondata", "pk": 108, "fields": {"action_name": "execute-qemu", "action_level": "2.1.1", "action_summary": "execute qemu to boot the image", "action_description": "call qemu to boot the image", "meta_type": 2, "testdata": 3, "yaml_line": null, "description_line": null, "log_section": null, "duration": null, "timeout": 30, "testcase": null, "count": null, "max_retries": 1}}, {"model": "lava_results_app.actiondata", "pk": 109, "fields": {"action_name": "auto-login-action", "action_level": "2.2", "action_summary": "Auto-login after boot with support for kernel messages.", "action_description": "automatically login after boot using job parameters and checking for messages.", "meta_type": 2, "testdata": 3, "yaml_line": null, "description_line": null, "log_section": null, "duration": null, "timeout": 30, "testcase": null, "count": null, "max_retries": 1}}, {"model": "lava_results_app.actiondata", "pk": 110, "fields": {"action_name": "expect-shell-connection", "action_level": "2.3", "action_summary": "Expect a shell prompt", "action_description": "Wait for a shell", "meta_type": 2, "testdata": 3, "yaml_line": null, "description_line": null, "log_section": null, "duration": null, "timeout": 30, "testcase": null, "count": null, "max_retries": 1}}, {"model": "lava_results_app.actiondata", "pk": 111, "fields": {"action_name": "export-device-env", "action_level": "2.4", "action_summary": "Exports environment variables action", "action_description": "Exports environment variables to the device", "meta_type": 2, "testdata": 3, "yaml_line": null, "description_line": null, "log_section": null, "duration": null, "timeout": 30, "testcase": null, "count": null, "max_retries": 1}}, {"model": "lava_results_app.actiondata", "pk": 112, "fields": {"action_name": "lava-test-retry", "action_level": "3", "action_summary": "Retry support for Lava Test Shell", "action_description": "Retry wrapper for lava-test-shell", "meta_type": 3, "testdata": 3, "yaml_line": null, "description_line": null, "log_section": null, "duration": null, "timeout": 30, "testcase": null, "count": null, "max_retries": 1}}, {"model": "lava_results_app.actiondata", "pk": 113, "fields": {"action_name": "lava-test-shell", "action_level": "3.1", "action_summary": "Lava Test Shell", "action_description": "Executing lava-test-runner", "meta_type": 3, "testdata": 3, "yaml_line": null, "description_line": null, "log_section": null, "duration": null, "timeout": 30, "testcase": null, "count": null, "max_retries": 1}}, {"model": "linaro_django_xmlrpc.authtoken", "pk": 1, "fields": {"secret": "kz8wyxmldwahe4w4086ceadedfwd0z7tadr87i60u1z30xymq38xy35ji98f0h6fgqmpwr3161zq87dytza70iqyhx5ab6xrzgh5lp1ghbcbrb0q650x8tpkgrm0a9n7", "description": "", "created_on": "2020-04-30T14:06:04.118Z", "last_used_on": null, "user": 2}}]
```

Comment:
Could you format this file to be human readable? I guess it would be useful in the future when writing tests

Reply:
This is dump from LAVA database. I'm not sure if editing this file is a good idea. At some point it might become obsolete if LAVA's db schema changes.

Reply:
Humm, makes sense. How to figure out what's in the database? I saw one test that you fetch a testjob with id = 3. I couldn't figure out where that came from.

Reply:
I tried reformatting it, but both jq and yq complain it's not valid JSON/YAML. I'll try something else.

Reply:
Ah, looks much better! Thanks!
I guess if we need more details on the job definition we could just copy and format it ourselves.